matter_server/client/client.py

```diff
@@ -53,16 +41,18 @@ class MatterClient:
 
     def __init__(self, ws_server_url: str, aiohttp_session: ClientSession):
         """Initialize the Client class."""
-        self.ws_server_url = ws_server_url
+        self.loop: asyncio.AbstractEventLoop | None = None
```

Comment:
I don't think the loop should be a feature for library users. Can we keep it protected?

Reply:
We may want it later in the sub controllers ?

Reply:
I'd rather get it there as needed than make it non protected attribute.